pkg/osbuild/vagrant_stage.go

```diff
@@ -12,10 +12,6 @@ type VagrantStageOptions struct {
 
 func (VagrantStageOptions) isStageOptions() {}
 
-func (o *VagrantStageOptions) validate() error {
-	return nil
-}
-
```

Comment:
Wouldn't it make sense to move the check from `NewVagrantStageOptions()` to this method instead?

Reply:
It does, I've changed it and force pushed.